style.css

```diff
@@ -0,0 +1,316 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+  background-color: #fff;
+}
+
+.my-logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.my-menu {
+  position: relative;
+  cursor: pointer;
+  user-select: none;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+a {
+  width: 117px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#introduction {
+  padding: 12px;
+  background-image: url('img/background.svg');
+  background-size: 94%;
+  background-repeat: no-repeat;
+  background-color: white;
+  border-bottom-left-radius: 70px;
+  height: 80vh;
+  width: 100%;
+}
+
+.socialnetworks {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 20px;
+}
+
+.socialnetworks1 {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 25px;
+}
+
+.socialnetworks2 {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 200px;
+  font-size: 24px;
+  color: #000;
+}
+
+.socialnetworks3 {
+  display: flex;
+  list-style: none;
+  padding-left: 0%;
+  gap: 290px;
+  font-size: 24px;
+  color: #000;
+}
+
+button {
+  border: 1px solid purple;
+  border-radius: 5px;
+  padding: 5px 20px;
+  background-color: white;
+  color: purple;
+  cursor: pointer;

```

Comment:
- [ ]  As mentioned by the previous reviewer please add a blue border to the button in order to stick to the Figma design.

## Your design 

![image](https://user-images.githubusercontent.com/95243996/205685062-55b43c37-c03a-4ad5-86b1-865d7ae788b2.png)

## Figma design
 
![image](https://user-images.githubusercontent.com/95243996/205685249-86677268-9f6f-4eac-acda-815bee3ae44d.png)